test/libraries/SafeCast.t.sol

```diff
@@ -31,8 +31,17 @@ contract SafeCastTest is Test {
         }
     }
 
+    function test_fuzz_toUint128(int128 x) public {
```

Comment:
can we rename these tests more explicity? now that we have 2 functions both called `toUint128`
`test_fuzz_toUint128_fromInt128`
and below
`test_toUint128_fromUint256`

and any other tests we have for those 2 functions